app/views/accounts/_phone.html.erb

```diff
@@ -10,7 +10,7 @@
       <div class="grid-row grid-gap-2">
         <div class="grid-col-fill">
           <%= PhoneFormatter.format(phone_configuration.phone) %>
-          <% if current_user.default_phone_configuration == phone_configuration %>
+          <% if current_user.default_phone_configuration == phone_configuration && MfaContext.new(current_user).phone_configurations.count > 1 %>
```

Comment:
Do we have an issue with phone showing default even if its just one? or do we think its not needed information.

Reply:
Based on the design, it was deemed as not needing that information